migrations/20240517094752_create_build.down.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
```

Comment:
I would expect that `sqlx` runs the migrations in a transaction by default?

Reply:
Ah, I didn't know that. Apparently it does, but only for Postgres.